src/platform/site-wide/user-nav/components/HelpMenu.jsx

```diff
@@ -33,6 +33,7 @@ class HelpMenu extends React.Component {
 
     return (
       <DropDownPanel
+        onClick="window.dataLayer.push({ 'event': 'nav-jumplink-click' });"
```

Comment:
Can we switch to `recordEvent(..)` here instead? It's just an alias to `window.data.Layer.push`

Reply:
You bet, @ncksllvn - updated :+1: